crates/lightbeam/Cargo.toml

```diff
@@ -11,24 +11,24 @@ keywords = ["webassembly", "wasm", "compile", "compiler", "jit"]
 edition = "2018"
 
 [dependencies]
-smallvec = "1.0.0"
+smallvec = "1.2.0"
 dynasm = "0.5.2"
-dynasmrt = "0.5.2"
+dynasmrt = "0.6.0"
 wasmparser = "0.51.2"
 memoffset = "0.5.3"
 itertools = "0.8.2"
 capstone = "0.6.0"
-thiserror = "1.0.9"
+thiserror = "1.0.11"
 cranelift-codegen = "0.59.0"
-multi_mut = "0.1"
+multi_mut = { git = "https://github.com/golddranks/multi_mut", rev = "561e9f7506e8aeddd7ee672d0f844437babe3da8" }
```

Comment:
This would prevent this crate from being published on crates.io.

Reply:
Thanks for commenting, @bjorn3. Yes, I've already figured this out. Looks like @Vurich had proposed better approach that would eliminate `multi_mut` completely.

Reply:
Cool, just to confirm, we do publish on crates.io, so we'll need the better approach here :slightly_smiling_face: .